shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.test.js

```diff
@@ -37,10 +45,49 @@ describe('generateStandingPretrialNoticeTemplate', () => {
 
     // TODO: Flesh this out further when template is done
     expect(result.indexOf('<!DOCTYPE html>')).toBe(0);
-    //expect(result.indexOf('Test Case Caption')).toBeGreaterThan(-1);
-    //expect(result.indexOf('123-45S')).toBeGreaterThan(-1);
-    //expect(result.indexOf('City')).toBeGreaterThan(-1);
-    //expect(result.indexOf('ST')).toBeGreaterThan(-1);
+    expect(result.indexOf('Test Case Caption')).toBeGreaterThan(-1);
+    expect(result.indexOf('123-45S')).toBeGreaterThan(-1);
+    expect(result.indexOf('City')).toBeGreaterThan(-1);
+    expect(result.indexOf('ST')).toBeGreaterThan(-1);
+    expect(result.indexOf('Guy Fieri')).toBeGreaterThan(-1);
+    expect(result.indexOf('STANDING PRETRIAL NOTICE')).toBeGreaterThan(-1);
+  });
+
+  it('Returns HTML with the given case and trial session data with an incorrect respondent structure', async () => {
```

Comment:
I believe the intention was to test for 100% branch coverage. In the code we have:
```
if (trialInfo.respondents && trialInfo.respondents.length) { ... }
```

Providing an object instead of an array will hit the `trialInfo.respondents.length` evaluation to give us that branch coverage. I'm updating the description and adding a comment that points this out in the test data.

Reply:
I'm also going to remove those //Todo  comments and flesh the test out a little more while I'm in there :)